### DISCUSSION TITLE
[A-L] (2023/24) Foglio 9 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#198] [Esercizi - Bertini] [answered]
![Screenshot 2023-11-27 134930](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/32908a56-bc68-424f-be9c-ccea28c63511)

**Comment by CiottoloMaggico** [comment, marked as answer]
## Soluzione confermata in classe
Analogo alla mia soluzione #90 
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/0fadf2cd-3c9f-46f1-a7d4-c7f309e0d348)

**Comment by Elia-Belli** [comment]
$$\int^{+\infty}\_{-\infty}xf_X(x)\ dx=^{int. per\ parti}x\cdot F_X(x)\Bigg|^{+\infty}\_{-\infty}-\int^{+\infty}\_{-\infty}F_X(x)\ dx$$

Ricordando che $F_X(x)=\mathbb{P}(X\leq x)=1-\mathbb{P}(X>x)$:
$$=x\cdot F_X(x)\Bigg|^{+\infty}\_{-\infty}-\int^{+\infty}\_{-\infty}1-\mathbb{P}(X>x)\ dx=x\cdot F_X(x)-x\Bigg|^{+\infty}\_{-\infty}+\int^{+\infty}\_{-\infty}\mathbb{P}(X>x)\ dx$$

$$\Bigg[ \lim_{x\to +\infty}[xF_X(x)-x] \Bigg] +\int^{+\infty}\_{-\infty}\mathbb{P}(X>x)\ dx$$

$$0+\int^{+\infty}\_{-\infty}\mathbb{P}(X>x)\ dx=^{(X\geq 0)}\int^{+\infty}_0\mathbb{P}(X>x)\ dx$$
